global/testModal.html

```diff
@@ -0,0 +1,52 @@
+<!-- Modal -->
+<div id="myModal" class="modal show" role="dialog">
+  <div class="modal-dialog">
+
+    <!-- Modal content-->
+    <div class="modal-content">
+      <div class="modal-header">
+        <button type="button" class="close" data-dismiss="modal">&times;</button>
+        <h4 class="modal-title">Modal Header</h4>
+      </div>
+      <div class="modal-body">
+        <p>Some text in the modal.</p>
+      </div>
+      <div class="modal-footer">
+        <button type="button" class="btn btn-default" data-dismiss="modal">Close</button>
+      </div>
+    </div>
+
+  </div>
+</div>
+
+
+
+
+
+<form role="form" method="post" action="<?php echo htmlspecialchars($_SERVER["PHP_SELF"]);?>">
+  <fieldset>
+      <div class="form-group">
+          <label>New Password</label>
+          <input class="form-control"
+          name="password1"
```

Comment:
Hey thats my password to my bank account

Reply:
oh cool thanks, what's your username?

Reply:
admin